tests/integration/targets/lookup_rabbitmq/tasks/ubuntu.yml

```diff
@@ -136,7 +136,7 @@
 - assert:
     that:
       - "rabbitmq_vhost_error is failed"
-      - "'NOT_ALLOWED' in rabbitmq_vhost_error.msg"
+      - ("'NOT_ALLOWED' in rabbitmq_vhost_error.msg") or ("'Connection issue' in rabbitmq_vhost_error.msg")
```

Comment:
If the connection issue coming up in the log is a valid return when it can't look up a host (as opposed to not being able to talk to `localhost:5672`) then this looks good to me. I'm trying to test it but currently failing on a previous test...

Reply:
It's a bit of a strange one... in the output of the tests for this PR (https://dev.azure.com/ansible/3505dd95-55ce-4e1a-b3f2-708a933091e4/_apis/build/builds/57609/logs/160) it returned below...

```
2022-10-19T23:10:30.8456419Z 01:38 [0;31mfatal: [testhost -> localhost]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'community.rabbitmq.rabbitmq'. Error was a <class 'ansible.errors.AnsibleError'>, original message: Connection issue: (530, 'NOT_ALLOWED - vhost missing not found')"}[0m
```
For some reason it's not picking up the NOT_ALLOWED anymore.